style.css

```diff
@@ -412,7 +412,12 @@ h4{
   margin-left: 5px;
```

Comment:
Maybe under the panel class, change the background color to something like `#fff1` so you can see the dropdown itself. Also looks like you need to change the color of the `+` and <code>&minus;</code> indicators to match the text's light gray.

Comparison with deadline vs rules:
![image](https://user-images.githubusercontent.com/54484616/132104552-06b0455f-882b-44e8-bb74-3bb0844e630c.png)

Reply:
@ethank2222